<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>364</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
	background:		rgba(35,31,32,255);
	border: none;/*6px solid rgba(0,174,239,255);*/
}

/*---------- QPushButton ---------*/
QPushButton
{
	border: 2px solid rgba(0,174,239,255);
	font: 7pt &quot;Corbel&quot;;
	color: white;
}

QPushButton:pressed
{
	border: 2px solid white;
	font: 7pt &quot;Corbel&quot;;
	background: rgba(0,174,239,255);
	color: white;
}


/* ---------- QLabels ------------*/
QLabel
{
	background:		rgba(0,0,0,0);
	color: 				white;
	font: 8pt &quot;Corbel&quot;;
	border:				none;
}

QLabel:hover
{
	border:				none;
}  

/* ---------- QSpinBox ------------*/
QSpinBox
{
	font: 8pt &quot;Corbel&quot;;
	border: 2px solid rgb(0,174,239);
	background-color: rgba(0, 0, 0, 0);
	color: white;
	padding:0,2, 0, 0;
}

QAbstractSpinBox:focus
{
	border: 2px solid rgb(255,255,255);
	outline: none;
}

QDoubleSpinBox
{
	font: 8pt &quot;Corbel&quot;;
	border: 2px solid rgb(0,174,239);
	background-color: rgba(0, 0, 0, 0);
	color: white;
	padding:0, 0, 0, 0;
}

/* ---------- QLineEdit ------------*/
QLineEdit
{
	font: 8pt &quot;Corbel&quot;;
	background: rgba(0,174,239,0);
	border: 2px solid rgba(0,174,239,255);
	border-radius: 0px;
	color: white;
	padding-left: 0px;
	padding-bottom: 0px;
}

QLineEdit:focus
{
	border: 2px solid rgb(255,255,255);
	outline: none;
}

QStackedWidget
{
border-top: 1px solid white;
background: black;
}

</string>
   </property>
   <widget class="QFrame" name="settingsFrame">
    <property name="geometry">
     <rect>
      <x>845</x>
      <y>130</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSpinBox" name="midiChannel">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>5</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::NoContextMenu</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>16</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="pedalCC">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>55</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="displayName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Corbel</family>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>EASY</string>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="midiChannelLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>MIDI Channel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sceneNameLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Display Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gainLabel_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Nav Pad CC#</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="gainLabel_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>89</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Pedal CC#</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="navPadCC">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::WheelFocus</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="maximum">
      <number>127</number>
     </property>
     <property name="value">
      <number>7</number>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="softstepLabel">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 28pt &quot;Futura-Normal&quot;;</string>
    </property>
    <property name="text">
     <string>SoftStep</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="connectedLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>51</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:8pt &quot;Futura-Normal&quot;;
color: red;</string>
    </property>
    <property name="text">
     <string>NOT CONNECTED</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sensitivityLabel">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>70</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Sensitivity</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QSpinBox" name="currentPreset">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>247</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="update">
    <property name="geometry">
     <rect>
      <x>845</x>
      <y>272</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*---------- QPushButton ---------*/
QPushButton
{
	border: 6px solid rgba(0,174,239,255);
	font: 12pt &quot;Corbel&quot;;
	color: white;
}

QPushButton:pressed
{
	background: rgba(0,174,239,255);
	border: 6px solid white;
	font: 12pt &quot;Corbel&quot;;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>SAVE + SEND</string>
    </property>
   </widget>
   <widget class="QLabel" name="currentPresetLabel">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>247</y>
      <width>89</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>CURRENT PRESET</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="backlightLabel">
    <property name="geometry">
     <rect>
      <x>844</x>
      <y>95</y>
      <width>89</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Backlight</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="sensitivity">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>70</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="minimum">
     <double>0.500000000000000</double>
    </property>
    <property name="maximum">
     <double>2.550000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="backlight">
    <property name="geometry">
     <rect>
      <x>945</x>
      <y>95</y>
      <width>24</width>
      <height>24</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
	padding-top: 1px;
}

QCheckBox::indicator
{
	width: 20px;
	height: 20px;
}

QCheckBox::indicator:unchecked
{
	image:url(:/resources/toggle_off.png)
}

QCheckBox::indicator:unchecked:hover
{
	image:url(:/resources/toggle_off.png)
}

QCheckBox::indicator:unchecked:pressed
{
	image:url(:/resources/toggle_on.png)
}

QCheckBox::indicator:checked
{
	image:url(:/resources/toggle_on.png)
}

QCheckBox::indicator:checked:hover
{
	image:url(:/resources/toggle_on.png)
}

QCheckBox::indicator:checked:pressed
{
	image:url(:/resources/toggle_off.png)
}

QCheckBox::indicator:unchecked:disabled
{
	image:url(:/resources/toggle_off_disabled.png)
}

QCheckBox::indicator:checked:disabled
{
	image:url(:/resources/toggle_off_disabled.png)
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="revert">
    <property name="geometry">
     <rect>
      <x>845</x>
      <y>327</y>
      <width>140</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">/*---------- QToolButton ---------*/
QToolButton
{
	border: 4px solid rgba(0,174,239,255);
	font: 8pt &quot;Corbel&quot;;
	color: white;
}

QToolButton:pressed
{
	background: rgba(0,174,239,255);
	border: 4px solid white;
	font: 8pt &quot;Corbel&quot;;
	color: white;
}</string>
    </property>
    <property name="text">
     <string>REVERT</string>
    </property>
   </widget>
   <zorder>settingsFrame</zorder>
   <zorder>softstepLabel</zorder>
   <zorder>connectedLabel</zorder>
   <zorder>sensitivityLabel</zorder>
   <zorder>backlightLabel</zorder>
   <zorder>sensitivity</zorder>
   <zorder>backlight</zorder>
   <zorder>currentPreset</zorder>
   <zorder>update</zorder>
   <zorder>currentPresetLabel</zorder>
   <zorder>revert</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>midiChannel</tabstop>
  <tabstop>sensitivity</tabstop>
  <tabstop>navPadCC</tabstop>
  <tabstop>pedalCC</tabstop>
  <tabstop>backlight</tabstop>
  <tabstop>displayName</tabstop>
  <tabstop>currentPreset</tabstop>
  <tabstop>update</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
